ciphey/basemods/Crackers/XandY.py

```diff
@@ -0,0 +1,108 @@
+# community
```

Comment:
Generally you should make a new PR for each feature, because Galactic failed CI which means it will take longer for this XandY feature to come through :)

Reply:
yeah, I thought that as well but noticed too late.